### PR TITLE
Updated README and Makefile

### PR DESCRIPTION
**Pull request by sophm256**
In the Makefile, I added make instructions to create a superuser and view logs. Also added seeing a preview of what the isort function would look like under the 'format-check'.

In the README, I did the following changes:
- Told the user to fork the repo, and then clone it.
- Removed the info about having errors after running the 'make migrate' command, because we no longer have errors (Thanks to @elmq0022 for ironing out those errors)
- Added info regarding 'make format' and 'make format-check'
- Added info regarding 'make shell' and 'make log'

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -52,6 +52,13 @@ format:
...
 format-check:
+	docker-compose exec web isort -rc -tc --atomic --diff .
 	docker-compose exec web black --diff .
 
 setup: setup_env build
+
+super: 
+	docker-compose exec web ./manage.py createsuperuser
+
+log:
```
Comment (raymondberg) [inline Makefile:63]: I think I prefer `tail-logs` here

File: Makefile [hunk 2]
```diff
@@ -52,6 +52,13 @@ format:
 	docker-compose exec web black .
 
 format-check:
+	docker-compose exec web isort -rc -tc --atomic --diff .
 	docker-compose exec web black --diff .
 
 setup: setup_env build
+
+super:
```
Comment (raymondberg) [inline Makefile:60]: Can we `superuser` this instead? Super is ambiguous.

File: Makefile [hunk 3]
```diff
@@ -52,6 +52,13 @@ format:
 	docker-compose exec web black .
 
 format-check:
+	docker-compose exec web isort -rc -tc --atomic --diff .
```
Comment (raymondberg) [inline Makefile:55]: This is a new one for me, thanks!

File: Makefile (hunk 2 above)
Comment (sophm256) [reply Makefile:60]: Sure. Will do.

File: Makefile (hunk 1 above)
Comment (sophm256) [reply Makefile:63]: Ok.

File: Makefile (hunk 3 above)
Comment (sophm256) [reply Makefile:55]: @elmq0022 has a current PR https://github.com/chicagopython/chipy.org/pull/285
that adds `docker-compose exec web isort -rc -tc --atomic .` to the Makefile 'format'. So I decided to add the accompanying diff command so people can preview.

**Comment by sophm256** [comment]
Thanks for reviewing my PR @raymondberg ! I appreciate your comments. 😄

**Pull request merged**